apps/shark_studio/web/ui/sd.py

```diff
@@ -392,6 +397,9 @@ def base_model_changed(base_model_id):
                         elem_id="negative_prompt_box",
                         show_copy_button=True,
                     )
+                    with gr.Row():
```

Comment:
Why do we add an explicit option to save the prompt? It should write prompts to the json config, so this seems a bit redundant to me.

Reply:
Yeah, the new config stuff is half-baked. Took it out, we can discuss how to handle it after we've got stable out.